app/controllers/TransferAgreementController.scala

```diff
@@ -3,12 +3,46 @@ package controllers
 import javax.inject.{Inject, Singleton}
 import org.pac4j.core.profile.CommonProfile
 import org.pac4j.play.scala.{Security, SecurityComponents}
+import play.api.data.Form
+import play.api.data.Forms._
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, Request}
 
 @Singleton
-class TransferAgreementController @Inject()(val controllerComponents: SecurityComponents) extends Security[CommonProfile] with I18nSupport  {
-  def transferAgreement(consignmentId: Long): Action[AnyContent] = Secure("OidcClient") { implicit request: Request[AnyContent] =>
-    Ok(views.html.transferAgreement(consignmentId))
+class TransferAgreementController @Inject()(val controllerComponents: SecurityComponents) extends Security[CommonProfile] with I18nSupport {
+
+  private val secureAction = Secure("OidcClient")
+  private val options: Seq[(String, String)] = Seq("Yes" -> "true", "No" -> "false")
+
+  val form = Form(
+    mapping(
+      "publicRecord" -> boolean
+        .verifying("Must answer yes", b => b),
+      "crownCopyright" -> boolean
+        .verifying("Must answer yes", b => b),
+      "english" -> boolean
+        .verifying("Must answer yes", b => b),
+      "digital" -> boolean
+        .verifying("Must answer yes", b => b),
+      "droAppraisalselection" -> boolean
+        .verifying("Must answer yes", b => b),
```

Comment:
You need another capital in `droAppraisalselection` There's a few places in the file where this is so I won't comment on them all

Reply:
fixed